packages/thrift-client/src/main/connections/axios.ts

```diff
@@ -19,10 +19,19 @@ export class AxiosConnection<TClient> extends HttpConnection<TClient> {
     this.request = requestApi
     this.request.defaults.responseType = 'arraybuffer'
     this.request.defaults.baseURL = `http://${this.hostName}:${this.port}`
+    this.request.defaults.headers = {
+      'host': this.hostName,
```

Comment:
Is this actually the host we want?

To make this go through linkerd, I needed to set "Host" to the service name. In my case, it was "content-build". By default, Request was setting "<host>:<port>".

The HTTP headers are actually defined capitalized:
* Host
* Connection
* Content-Type

Reply:
Oh, okay. I'll remove host and let consumers set it.

Reply:
I think the old Thrift client took the service as a parameter. It is nice to require it in code if it is required to work.